handMadeCalendarAdvance/MonthlyCalendarViewController.swift

```diff
@@ -261,26 +261,26 @@ extension MonthlyCalendarViewController: UICollectionViewDelegate {
     }
 }
 
-// MARK: - UIScrollViewDelegate
+// MARK: - UICollectionViewDelegateFlowLayout
+
+extension MonthlyCalendarViewController: UICollectionViewDelegateFlowLayout {
 
-extension MonthlyCalendarViewController: UIScrollViewDelegate {
-    
     //セルのサイズを設定
-    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAtIndexPath indexPath: IndexPath) -> CGSize {
+    func collectionView(_ collectionView: UICollectionView, layout collectionViewLayout: UICollectionViewLayout, sizeForItemAt indexPath: IndexPath) -> CGSize {
```

Comment:
__[SwiftLint]__ Line should be 120 characters or less: currently 160 characters

<sub>posted by Sider</sub>